src/Login.cpp

```diff
@@ -0,0 +1,141 @@
+#include "Login.hpp"
+
+Login::Login() { /* does nothing */
+}
+
+Login::~Login() {
+    /* no deletion required*/
+    outputCSV();
```

Comment:
We had talked about not calling printouts in the destructor in favor of explicit prints.

Reply:
moved the statement into userLogin in IM